server/player.lua

```diff
@@ -341,6 +341,7 @@ function QBCore.Player.CreatePlayer(PlayerData, Offline)
                 end
             end
         end
+        if self.PlayerData.money[moneytype] - amount <= QBConfig.Money.MinusLimit then return false end
```

Comment:
Looks good to me, I'd probably just make it `<` rather than `<=` to comply with the wording of 'maximum', as this would only be true for >-4999.

Very minor change though and is just to be consistent with the comment.

Reply:
made the change and contributed

Reply:
Thank you, I'm sorry to make you add to it again but we tend to use `QBCore.Config` to reference values rather than `QBConfig` just to prevent confusion and keep things consistent with QB resources outside of the core. It's another one of those "Consistency" things, not sure how I missed it the first time. This would just be for `server/player.lua:344` of course.

I do like this feature as it essentially simulates a proper overdraft.

Reply:
should be good to go!